src/assets/javascripts/components/search/query/index.ts

```diff
@@ -106,14 +106,6 @@ export function watchSearchQuery(
         history.replaceState({}, "", `${url}`)
       })
 
-  /* Set query from parameter */
```

Comment:
I think it is fine to keep it, and instead do the removal I mentioned, as that will minimize merge conflicts in the future.